scripts/normalize-by-median.py

```diff
@@ -204,12 +206,14 @@ def get_parser():
     parser.add_argument('--force', default=False, action='store_true',
                         help='Overwrite output file if it exists')
     add_loadhash_args(parser)
+    add_output_compression_type(parser)
     return parser
 
 
 def main():  # pylint: disable=too-many-branches,too-many-statements
     info('normalize-by-median.py', ['diginorm'])
     args = get_parser().parse_args()
+    enable_output_compression(args)
```

Comment:
see, here it will only work if you specify a -o, which is good, but it'd be nice if this worked for any and all output files no?